modules/canvaskit/canvaskit/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bluescape/canvaskit-wasm",
-  "version": "0.20.0-bluescape-2.2",
+  "version": "0.20.0-bluescape-2.3",
```

Comment:
Perhaps even 3.0 :)

Reply:
I'm not sure why it should be - it's API-compatible with 2.2, because it only adds a new function, it doesn't remove anything?
I thought updating the major version should be something that breaks compatibility?

Reply:
OK. You might be right.